app/jobs/event_job.rb

```diff
@@ -5,14 +11,52 @@ class EventJob
   include Hydra::AccessControlsEnforcement
   include SufiaHelper
 
+  # queue to runthe job on
   def queue_name
     :event
   end
 
-  attr_accessor :generic_file_id, :depositor_id
+  attr_accessor :depositor_id
 
-  def initialize(generic_file_id, depositor_id)
-    self.generic_file_id = generic_file_id
+  # @param the id of the user to create the event for
+  def initialize(depositor_id)
     self.depositor_id = depositor_id
```

Comment:
This isn't a change you've made, but I wonder if this should be `@depositor_id` instead of `self.depositor_id` to be more idiomatic.